docs/source/configuration/overview.mdx

```diff
@@ -578,6 +578,170 @@ The default value of `experimental_parallelism` is `1`.
 
 In practice, you should tune `experimental_parallelism` based on metrics and benchmarks gathered from your router.
 
+<MinVersion version="1.49.0">
+
+### Enhanced operation signature normalization
+
+</MinVersion>
+
+<ExperimentalFeature />
+
+<EnterpriseFeature />
```

Comment:
We only updated this recently in the field guide, but the signature normalisation functionality is not enterprise-only.